eos-updater-flatpak-installer/eos-updater-flatpak-installer-fallback.service.in

```diff
@@ -0,0 +1,59 @@
+[Unit]
+Description=Endless OS Fallback Flatpak Installer
+Documentation=man:eos-updater-flatpak-installer(8)
+# /home is a symlink to /var/home; /var/home is a symlink to /sysroot/home. The
+# second symlink is created by systemd-tmpfiles. Since we use ProtectHome=yes,
+# we must explicitly order this unit after tmpfiles are created.
+Requires=local-fs.target eos-extra-settled.target systemd-tmpfiles-setup.service
+After=local-fs.target eos-extra-settled.target systemd-tmpfiles-setup.service
+ConditionKernelCommandLine=!eos-updater-disable
+DefaultDependencies=no
+Conflicts=shutdown.target
+
+# Unlike eos-updater-flatpak-installer.service, this service can run outside
+# of the normal update process.
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=@libexecdir@/eos-updater-flatpak-installer --pull --mode=perform
```

Comment:
Could update the comment on line 259 of `eos-updater-flatpak-installer/main.c` now that the `--pull` option is being used here.

Reply:
Good catch! I have no idea how you spotted that. Fixed in a separate commit.